packages/base/src/UI5Element.js

```diff
@@ -58,6 +58,7 @@ class UI5Element extends HTMLElement {
 
 		this._monitoredChildProps = new Map();
 		this._firePropertyChange = false;
+		this.shouldInvalidateParent = false;
```

Comment:
The convention in this file is all private members should be with _, so: `_shouldInvalidateParent `